pkg/codegen/nodejs/gen_program_expressions.go

```diff
@@ -703,7 +703,7 @@ func (g *generator) GenLiteralValueExpression(w io.Writer, expr *model.LiteralVa
 	case model.BoolType:
 		g.Fgenf(w, "%v", expr.Value.True())
 	case model.NoneType:
-		g.Fgen(w, "undefined")
```

Comment:
None in PCL is "null" not "undefined". undefined in Node really is "not present" and we elide them from stack outputs specifically. This _should_ eventually work to return null as a stack output though.